mmhuman3d/models/losses/prior_loss.py

```diff
@@ -65,6 +67,99 @@ def forward(self,
         return shape_prior_loss
 
 
+@LOSSES.register_module()
+class LimbLengthLoss(nn.Module):
+    """Limb length loss for body shape parameters. Note that it should take
```

Comment:
May add a bit more details on the purpose of this loss. e.g. This loss penalizes changing limb lengths.

Reply:
Details have been added in https://github.com/open-mmlab/mmhuman3d/pull/102/commits/db604f492a8eff72dbb06ce6ad50098dbb61ccd6